ui/src/containers/Search/index.tsx

```diff
@@ -1,19 +1,39 @@
 import { TextInput } from '@patternfly/react-core';
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { useMst } from '../../store/root';
+import { UseDebounce } from './useDebounce';
 import './Search.css';
 
 const Search: React.FC = () => {
   const { resources } = useMst();
-
   const [value, setValue] = useState('');
+  let url = '';
```

Comment:
since we are not using `url` anywhere except for line 31. Can we declare this there only?